src/algorithms/paramspacesgd/paramspacesgd.jl

```diff
@@ -1,68 +1,7 @@
 
```

Comment:
@Red-Portal, this file can be renamed to `interface.jl`, to keep the `step` function for parameter space VI and new ones that you want to add.